templates/express-starter/app.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const express = require('express');
+const express = require('express'); // eslint-disable-line
```

Comment:
We should point specific rule in disabler (as e.g. here: https://github.com/serverless-components/express/blob/fc2544d568c921fa97c7cc5b600063f56907bdba/src/serverless.js#L3 )

Otherwise we may accidentally hide important error (e.g. typo in function name)

Reply:
done